src/main/java/seedu/address/model/item/Quantity.java

```diff
@@ -9,11 +9,12 @@
  */
 public class Quantity {
 
-
     public static final String MESSAGE_CONSTRAINTS =
             "Quantity should only contain numbers, and it should be at least 1 digit long";
     public static final String VALIDATION_REGEX = "\\d+";
     public final String value;
+    // for ease of access
+    public final int number;
```

Comment:
don't do this, it creates coupling btwn the variables - use a method that returns `Integer.parseInt(...)`